### PR TITLE
REFACTOR: changed ethernet image-SVG

### PR DESCRIPTION
**Pull request by RazeSevi**
I changed the ethernet image with a modern touch.

### CODE REVIEW

**Review by F0xedb — CHANGES_REQUESTED** [review]
This commit is using a svg image that comes from mediawiki (wikipedia)
I would like to say a few things:

- Make your own svg images
- Make a `dark` variant for this image
- Make a `lan_off` variant for this image
- reduce the filesize by removing extra whitespace

**Review by F0xedb — CHANGES_REQUESTED** [review]
I would like 2 more changes to be applied to this patch

image

First of all the lines should be thicker, as that makes it more readable (look at the wifi icon vs the lan icon)

Secondly your file sizes are way to big:
image
**100 - 300 KB is way to big**
Please reduce the complexity of the file to 1-2KB as you are introducing to much overhead to the system

TL;DR
1. make the lines thicker for easier readability
2. reduce the filesize (all my other svg's combined take up less space)

If those are applied then this will be merged!

**Comment by F0xedb** [comment]
PS: a general tip when developing svg's (don't use paths as they tend to consume a lot of data)
You should work with primitives such as boxes, rectangles, arcs, triangles, etc
Instead of using the path/draw/paint tools

**Review by F0xedb — APPROVED** [review]
